service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderProgressionServiceTest.java

```diff
@@ -50,11 +54,20 @@
     @Autowired
     private ObjectMapper mapper;
 
+    @Mock
+    private ApplicationEventPublisher eventPublisher;
+
+    @Mock
+    private RequestData requestData;
+
     private CaseManagementOrderProgressionService service;
 
     @BeforeEach
     void setUp() {
-        this.service = new CaseManagementOrderProgressionService(mapper);
+        given(requestData.authorisation()).willReturn("Bearer");
```

Comment:
Are these 2 lines needed?

Reply:
Also I know you haven't touched the method but `caseDataWithCaseManagementOrder` doesn't need to throw an IOException